### ISSUE TITLE
Head titles are not correct for some pages

### ISSUE THREAD

**Issue by arslanashraf7** [mitodl/mitxonline#148]
### Steps to Reproduce
1) Visit any of them (https://rc.mitxonline.mit.edu/,  https://rc.mitxonline.mit.edu/courses/course-v1:MITx+14.100x/)
2) Visit any of them (https://rc.mitxonline.mit.edu/signin/, https://rc.mitxonline.mit.edu/create-account/)
3) Take a look at the head title(Page Title in the browser tab)

Take a look at the screenshots below for clarity.

### Expected Behavior
- Tab/Head title should be correct as per the URL being visited (e.g. https://xpro-rc.odl.mit.edu/)

### Actual Behavior
- Head title is not displayed correctly for wagtail pages e.g. `Home, Product Detail`

**Implementation Detail:**
I believe this odd behavior is only shown on the pages that are based on wagtail pages using Django templates. And the potential solution to fix this would be to add/fix the context processor with proper `settings.SITE_NAME`. For `Context Processor` take a look at https://github.com/mitodl/mitxonline/pull/140/files#

### Screenshot or Screencast
<img width="772" alt="Screenshot 2021-09-03 at 5 30 32 PM" src="https://user-images.githubusercontent.com/34372316/132006711-431493c9-faa3-4514-8659-f793bfcc7d59.png">
<img width="991" alt="Screenshot 2021-09-03 at 5 30 48 PM" src="https://user-images.githubusercontent.com/34372316/132006732-ca98ef91-840c-410b-828e-df8002a9dac8.png">
<img width="1063" alt="Screenshot 2021-09-03 at 5 31 08 PM" src="https://user-images.githubusercontent.com/34372316/132006735-21ff530a-ca56-43c1-8b01-74046b461c14.png">

**Comment by pdpinch** [comment]
This was also flagged in our accessibility review, so I'm adding the accessibility label.

**Comment by arslanashraf7** [comment]
Closed via #152
